c++/src/capnp/rpc-prelude.h

```diff
@@ -87,6 +87,16 @@ class RpcSystemBase {
 
   void setTraceEncoder(kj::Function<kj::String(const kj::Exception&)> func);
 
+  kj::Promise<void> run();
+  // Listens for incoming RPC connections and handles them. Never returns normally, but could throw
```

Comment:
So, `rpc-prelude.h` is supposed to be private implementation details. This doc comment should go in `rpc.h` instead. I guess you can declare `run()` there with inline implementation `{ return RpcSystemBase::run(); }`, just for the sake of putting a doc comment on it...